tests/Backend/Snowflake/CloneInputMappingTest.php

```diff
@@ -0,0 +1,312 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Test\Backend\Snowflake;
+
+use Keboola\Csv\CsvFile;
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\Exception;
+use Keboola\StorageApi\Workspaces;
+use Keboola\Test\Backend\Workspaces\Backend\WorkspaceBackendFactory;
+use Keboola\Test\Backend\Workspaces\WorkspacesTestCase;
+
+class CloneInputMappingTest extends WorkspacesTestCase
```

Comment:
`WokspaceLoadCloneTest` / `CloneIntoWorkspaceTest`? Metody testů pak clone ani input mapping nutně obsahovat nemusí, je to v názvu classy.

Reply:
přejmenoval jsem